src/components/Routes.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {Route, Switch, Router, Redirect} from 'react-router-dom'
+import {HomePage} from '../components'
+import {history} from '../utils'
+
+const Routes = function (props) {
+  return (
+    <Router history={history}>
+      <Switch>
+        <Route exact path="/" component={HomePage} />
```

Comment:
I like that you set up routes here, though it currently looks like a single page app. This isn't necessary unless you have plans to expand your app in the future.

Reply:
Boilerplate